src/components/Admin/Menu.js

```diff
@@ -51,7 +54,7 @@ class Menu extends Component {
             
             <div className="dropdown">
               <a className="nav-link dropdown-toggle" href="/a" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                Welcome, User !
+                Welcome, <span style={{ color: 'red' }}>{userName}</span> !
```

Comment:
User Name is styled color red anytime?

Reply:
Yes. We can only access to admin page after logged in by admin acccount. So,only admin name is showed with red color